watertap/flowsheets/electroNP/electroNP_flowsheet.py

```diff
@@ -164,27 +164,13 @@ def build_flowsheet():
     # scaling
     for var in m.fs.component_data_objects(pyo.Var, descend_into=True):
         if "flow_vol" in var.name:
-            iscale.set_scaling_factor(var, 1e3)
+            iscale.set_scaling_factor(var, 1e2)
         if "temperature" in var.name:
-            iscale.set_scaling_factor(var, 1e-1)
+            iscale.set_scaling_factor(var, 1e-2)
         if "pressure" in var.name:
-            iscale.set_scaling_factor(var, 1e-3)
-        if "alkalinity" in var.name:
-            iscale.set_scaling_factor(var, 1e-1)
+            iscale.set_scaling_factor(var, 1e-5)
         if "conc_mass_comp" in var.name:
-            iscale.set_scaling_factor(var, 1e1)
-        if "conc_mass_comp[S_IN]" in var.name:
-            iscale.set_scaling_factor(var, 1e-1)
-        if "conc_mass_comp[S_IP]" in var.name:
-            iscale.set_scaling_factor(var, 1e-1)
-        if "conc_mass_comp[S_PO4]" in var.name:
-            iscale.set_scaling_factor(var, 1e-1)
-        if "conc_mass_comp[S_NH4]" in var.name:
-            iscale.set_scaling_factor(var, 1e1)
-        if "conc_mass_comp[S_F]" in var.name:
-            iscale.set_scaling_factor(var, 1e-1)
-        if "conc_mass_comp[X_I]" in var.name:
-            iscale.set_scaling_factor(var, 1e-1)
+            iscale.set_scaling_factor(var, 1e3)
```

Comment:
I would say it might be risky to change the concentration scaling factor to 1e3 (can be high for some species), can you clarify the condition number after applying the new scaler?

Reply:
To be honest, this was the only way I could get the flowsheet to solve. Otherwise, the AD failed to initialize. I didn't spend too much time trying to refine this flowsheet since it wasn't the focus of this PR, but I'm sure you could get it solving w/o changing this scaling factor - it would just take time.